osu.Game.Tests/Visual/Gameplay/TestScenePlayerLoader.cs

```diff
@@ -240,6 +243,15 @@ private void addVolumeSteps(string volumeName, Action beforeLoad, Action afterLo
             AddUntilStep("wait for player load", () => player.IsLoaded);
         }
 
+        [TestCase(true)]
+        [TestCase(false)]
+        public void TestEpilepsyWarning(bool warning)
```

Comment:
This test should be checking whether the epilepsy warning is visible (by finding it via `this.ChildrenOfType` and inspecting its `Alpha`).

Also not sure whether this is the best place for it. Thus far `BeatmapMetadataDisplay` didn't really have actual logic so a test scene was maybe a bit extravagant, but in this case it would be certainly much easier to test in a new separate scene.